Library/Environment.swift

```diff
@@ -117,11 +126,14 @@ public struct Environment {
     koala: Koala = Koala(client: KoalaTrackingClient(endpoint: .production)),
     language: Language = Language(languageStrings: Locale.preferredLanguages) ?? Language.en,
     launchedCountries: LaunchedCountries = .init(),
+    lightImpactFeedbackGenerator: UIImpactFeedbackGeneratorType = UIImpactFeedbackGenerator(style: .light),
```

Comment:
Given that at the callsite we access all of these through an extension on `UIFeedbackGenerator`, would it not be less to add to the `Environment` (and less to mock) if we just added a type like `UIFeedbackGeneratorType`?

I guess one other question I have is - does it matter if these generator instances exist for the lifetime of the app? I'm guessing they're pretty lightweight like tap gestures and it doesn't matter too much, but perhaps if we moved the instantiation of them into the type that we create then they'd only exist when we need them.

Lastly - I've seen in the documentation that we're supposed to call [prepare()](https://developer.apple.com/documentation/uikit/uifeedbackgenerator/2369818-prepare), I don't think we were doing that before, but perhaps it's a good idea to add it.